packages/libs/eda/src/lib/core/components/visualizations/InputVariables.tsx

```diff
@@ -253,11 +253,7 @@ export function InputVariables(props: Props) {
                             : undefined
                         }
                       >
-                        {input.label +
```

Comment:
Thank you for your review @asizemore 👍 The reason why I removed the term, "(fixed)", was based on Danica's speaking in the following. If you think it should be in there, then I will get it back.

at https://epvb.slack.com/archives/C0132PM1MT5/p1668802161518429

Danica
i think this makes sense, for consistency with other plots. Maybe don’t include “(fixed)” but put “phylum” in the box and don’t have the dropdown. I think it will be obvious that it is fixed

Reply:
Ahh I see. Thanks for the link! I think we should add it back in. The "(fixed)" gives some indication of why that entry looks different from the others, since we don't have any styling difference between the name of the input (ex "Y-axis") and the fixed variable (ex "Abundance").

Reply:
Thank you for your detailed information @asizemore 👍 It makes sense to me and I reverted it back as following screenshot: also had to change something else to display it better.

![1490 mbio abundance input02](https://user-images.githubusercontent.com/12802305/231816338-45f412e2-e2de-435e-857c-296d1ddf9f3c.png)